dynawo/3rdParty/CMakeLists.txt

```diff
@@ -521,11 +546,18 @@ else()
 
       CONFIGURE_COMMAND   <SOURCE_DIR>/bootstrap.sh
                           --without-icu
-                          --with-libraries=filesystem,program_options,serialization,system,log,iostreams
+                          ${TmpVar}
 
       BUILD_COMMAND       <SOURCE_DIR>/b2
                           -j${NUMBER_OF_PROCESSORS}
-                          -sNO_BZIP2=1 cxxflags=-fPIC
+                          -sNO_BZIP2=1
+                          cxxflags=-fPIC
+                          --with-test
+                          --with-date_time
+                          --with-filesystem
```

Comment:
you said you were removing those --with-X yet there are still here

Reply:
little mistake, just removed :)